### PR TITLE
Update to Support New chain

### PR DESCRIPTION
**Pull request by molecula451**
OORT Mainnet Dev (Testnet)

ref: https://oortech.com

---

## PR-Codex overview
This PR focuses on adding support for the OORT MainnetDev (Testnet) chain. 

### Detailed summary
- Added `oortMainnetDev` chain definition in `src/chains/definitions/oortmainnetDev.ts`
- Updated `src/chains/index.ts` to export `oortMainnetDev` definition

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by molecula451** [comment]
the chain is available on: https://github.com/ethereum-lists/chains

https://chainlist.org/?testnets=false&search=oort
https://chainid.network

File: src/chains/index.ts [hunk 1]
```diff
@@ -194,6 +194,7 @@ export { zilliqaTestnet } from './definitions/zilliqaTestnet.js'
 export { zora } from './definitions/zora.js'
 export { zoraSepolia } from './definitions/zoraSepolia.js'
 export { zoraTestnet } from './definitions/zoraTestnet.js'
+export { oortMainnetDev } from './definitions/oort.js'
```
Comment (jxom) [inline src/chains/index.ts:197]: Please alphabetize this.

**Review by jxom — CHANGES_REQUESTED** [review]
Please add a changeset.

https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#4-add-changeset

File: src/chains/index.ts (hunk 1 above)
Comment (molecula451) [reply src/chains/index.ts]: thanks friend @jxom done at: https://github.com/wevm/viem/pull/1743/commits/aa07d97d8f25a735fb384786cb67780662f2fe47

**Comment by molecula451** [comment]
@jxom https://github.com/wevm/viem/pull/1743/commits/b3537df2b53cf0a5dc8246ccde2d4c50d2a0d2dc

File: src/chains/definitions/oort.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { defineChain } from '../../utils/chain/defineChain.js'
+
+export const oortMainnetDev = /*#__PURE__*/ defineChain({
+  id: 9584236,
```
Comment (jxom) [inline src/chains/definitions/oort.ts:4]: Is this meant to be 9700 as per chainlist?

File: src/chains/index.ts (hunk 1 above)
Comment (jxom) [reply src/chains/index.ts]: ty friend <3

File: src/chains/definitions/oort.ts (hunk 2 above)
Comment (molecula451) [reply src/chains/definitions/oort.ts]: yes correct chainId is 9700 https://github.com/wevm/viem/pull/1743/commits/363dc4abeffcb66f43abf12b7877e51ba3769663

**Comment by molecula451** [comment]
can we merge this @jxom ?

**Comment by jxom** [comment]
yes we can

**Comment by molecula451** [comment]
stay tuned I will sent PR for the actual mainnet

**Pull request merged**